client/dive-common/components/ControlsContainer.vue

```diff
@@ -114,6 +118,17 @@ export default defineComponent({
           >
             Events
           </v-btn>
+          <v-btn
+            class="ml-2"
+            :class="{'timeline-button':currentView!=='Groups' || collapsed}"
+            depressed
+            :outlined="currentView==='Groups' && !collapsed"
+            x-small
+            tab-index="-1"
+            @click="toggleView('Groups')"
+          >
+            Groups
+          </v-btn>
```

Comment:
It will need to be unconditional in the next PR, otherwise there will be no way to add new group annotations.  I've de-emphasized the group features in response to other comments -- `TypeThresholds` is the new panel default.